pkg/reconcilermanager/controllers/rootsync_controller_test.go

```diff
@@ -136,6 +136,7 @@ func setupRootReconciler(t *testing.T, objs ...client.Object) (*syncerFake.Clien
 		fakeClient,
 		controllerruntime.Log.WithName("controllers").WithName("RootSync"),
 		s,
+		false,
```

Comment:
Add unit test for the rootsync controller as well.

Reply:
Root sync controller test has been added, follows the same structure with the repo sync test